src/core/less/variables/spacings.less

```diff
@@ -0,0 +1,259 @@
+// Spacings setup
```

Comment:
i think all of these variables should be set to 0 (except where necessary for bare structure)

Reply:
Done.